shopfloor/tests/test_cluster_picking_scan_line.py

```diff
@@ -249,6 +300,7 @@ def test_scan_line_location_error_several_package(self):
         # Although, if one line is already processed,
         # we can work automatically on the next one
         line.qty_done = line.product_uom_qty
+        line._action_done()
```

Comment:
Calling this method at the `stock.move.line` level seems wrong, what was the intent?